pages/dkp/kommander/2.1/security/policy-controls/gatekeeper/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: layout.pug
+navigationTitle: Enforce policies using Gatekeeper
+title: Enforce policies using Gatekeeper
+menuWeight: 24
+excerpt: Learn how to enforce policies using Gatekeeper
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD024 -->
+
+[Gatekeeper][gatekeeper] is the policy controller for Kubernetes, allowing organizations to enforce configurable policies using the [Open Policy Agent][opa], a policy engine for Cloud Native environments hosted by CNCF as a graduated-level project.
+
+This tutorial describes how to use Gatekeeper to enforce policies by rejecting non-compliant resources. Specifically, this tutorial describes two constraints as a way to show how to use Gatekeeper as an alternative to [Pod Security Policies][psp]:
+
+- [Prevent the running of privileged pods](#prevent-privileged-pods)
+- [Prevent mounting host path volumes](#prevent-host-path-volumes)
+
+## Before you begin
+Before starting this tutorial, verify the following:
+
+- You must have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You must have a properly deployed and running cluster. For information about deploying Kubernetes with default settings on different types of infrastructures, see the [Choose Infrastructure][choose-infrastructure].
+
+- Ensure Gatekeeper is [enabled](../../../install/configuration/http-proxy/#enable-gatekeeper)
+
+## Use Gatekeeper
+
+Gatekeeper uses the [OPA Constraint Framework][opa-constraints] to describe and enforce policy. Before you can define a constraint, you must first define a `ConstraintTemplate`, which describes both the [`Rego`][opa-rego] (a powerful query language) that enforces the constraint and the schema of the constraint. The schema of the constraint allows an admin to fine-tune the behavior of a constraint, much like arguments to a function.
+
+The Gatekeeper repository includes a [library of policies][gatekeeper-psp] to replace Pod Security Policies which we will use in the following tutorials.
```

Comment:
This link needs to be updated to ...  I'm pretty sure this one `[gatekeeper-library]` - which goes here: 
https://github.com/open-policy-agent/gatekeeper-library/tree/master/library/pod-security-policy

```suggestion
The Gatekeeper repository includes a [library of policies][gatekeeper-library] to replace Pod Security Policies which we will use in the following tutorials.
```

Reply:
done